backend/btrixcloud/profiles.py

```diff
@@ -155,12 +155,25 @@ async def commit_to_profile(
         self,
         browser_commit: ProfileCreate,
         storage: StorageRef,
+        user: User,
         metadata: dict,
-        profileid: Optional[UUID] = None,
+        existing_profile: Optional[Profile] = None,
     ) -> dict[str, Any]:
         """commit profile and shutdown profile browser"""
-        if not profileid:
+        # pylint: disable=too-many-locals
+
+        now = datetime.utcnow().replace(microsecond=0, tzinfo=None)
+
+        if existing_profile:
+            profileid = existing_profile.id
+            created = existing_profile.created
+            created_by = existing_profile.createdBy
+            created_by_name = existing_profile.createdByName
+        else:
             profileid = uuid4()
+            created = now
```

Comment:
Since we can't really do a migration for existing data, should we try to fill in the created* fields when updating the profile if they're previously null? Eg. when you save an existing profile with no creation data, it'll set both created and modified? Or leave as null?

Reply:
Ah, that's a good call! They should have a `created` date already, but we can set the related user fields on updates. Will push a commit with this.

Reply:
Hm, actually I guess we don't have reliable data for this, so might be better to leave `createdBy` and `createdByName` null rather than speculating?

`created` should be there already, it was already being asserted in the tests and added in the endpoint to commit a profile.

Reply:
yeah, and the other option would be to set the created to same as modified here, but not sure if that's better.

Reply:
I think it's probably better to leave it null? Otherwise it's potentially misleading information.